test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs

```diff
@@ -0,0 +1,245 @@
+
+        mWindow.Add(mSceneView);
+    }
+    private void SetupSceneViewCamera(SceneView sceneView)
+    {
+        // Default camera setting
+        // Note : SceneView always have 1 default camera.
+        Tizen.NUI.Scene3D.Camera defaultCamera = sceneView.GetCamera(0u);
+
+        defaultCamera.PositionX = 0.0f;
+        defaultCamera.PositionY = 0.0f;
+        defaultCamera.PositionZ = 5.55f;
+        defaultCamera.NearPlaneDistance = 0.5f;
+        defaultCamera.FarPlaneDistance = 50.0f;
+        //defaultCamera.Orientation = new Rotation(new Radian(new Degree(180.0f)), Vector3.YAxis);
+        defaultCamera.FieldOfView = new Radian(new Degree(45.0f));
+        defaultCamera.OrthographicSize = 2.7f;
+
+        // Additional camera setting (top view camera).
+        Tizen.NUI.Scene3D.Camera camera = new Tizen.NUI.Scene3D.Camera()
+        {
+            PositionX = 0.0f,
+            PositionY = -3.95f,
+            PositionZ = 0.0f,
+            NearPlaneDistance = 0.5f,
+            FarPlaneDistance = 50.0f,
+            // Rotate by XAxis first, and then rotate by YAxis
+            Orientation = new Rotation(new Radian(new Degree(90.0f)), Vector3.YAxis) *
+                            new Rotation(new Radian(new Degree(-90.0f)), Vector3.XAxis),
+            FieldOfView = new Radian(new Degree(70.0f)),
+            OrthographicSize = 2.7f,
+        };
+        sceneView.AddCamera(camera);
+    }
+
+    protected void CreateModel(string modelUrl)
+    {
+        // Release old one.
+        if(mModel != null)
+        {
+            mModel.Unparent();
+            mModel.Dispose();
+        }
+
+        mModel = new Model(MODEL_DIR + modelUrl)
+        {
+            Name = modelUrl,
+        };
+        mModel.ResourcesLoaded += (s, e) =>
+        {
+            Model model = s as Model;
+
+            // You can play animation if it has.
+            if(model.GetAnimationCount() > 0u)
+            {
+                model.GetAnimation(0u).Looping = true;
+                model.GetAnimation(0u).Play();
+            }
+            Tizen.Log.Error("NUI", $"{model.Name} size : {model.Size.Width}, {model.Size.Height}, {model.Size.Depth}\n");
+        };
+        mModelRotateAnimation = new Animation(modelRotateAnimationDurationMilliSeconds);
+        mModelRotateAnimation.AnimateBy(mModel, "Orientation", new Rotation(new Radian(new Degree(360.0f)), Vector3.YAxis));
+
+        mModelRotateAnimation.Looping = true;
+        mModelRotateAnimation.Play();
+
+        mSceneView.Add(mModel);
+    }
+
+    void SetupIBLimage(string specularUrl, string diffuseUrl, float iblFactor)
+    {
+        mSceneView.SetImageBasedLightSource(IMAGE_DIR + specularUrl, IMAGE_DIR + diffuseUrl,iblFactor);
+    }
+
+    void OnKeyEvent(object source, Window.KeyEventArgs e)
+    {
+        if (e.Key.State == Key.StateType.Down)
+        {
+            FullGC();
+            //Streamline.AnnotateChannelEnd(0);
+
+            switch( e.Key.KeyPressedName )
```

Comment:
Please follow C# coding style :
```suggestion
            switch (e.Key.KeyPressedName)
```

Reply:
Done.